scalableminds/webknossos
# Showing welcome header when there are no organisations

This PR shows the welcome header at `/` when no user is logged in and there are no existing organizations. It also disables the auto redirect to `onboarding` when entering there are no organizations.

### URL of deployed dev instance (used for testing):
- [https://showwelcomeheader.webknossos.xyz](https://showwelcomeheader.webknossos.xyz)

### Steps to test:
- I suggest testing it locally
- go to application.conf in the conf folder and set `insertInitialData = true` to `false`
- refresh the schema and then start the server
=> the path `/` should show the welcome header even when `allowOrganizationCreation` is set to false.
And there should be no redirect to `onboarding`. 

### Issues:
- fixes #3118

------
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Updated [documentation](../blob/master/docs) if applicable~~~
- ~~~[ ] Needs datastore update after deployment~~~
